packages/client/src/components/chat/beholder/paperdoll.css

```
@@ -0,0 +1,1004 @@
+    text-overflow: clip;
+}
+/* Disengaged: a thin overstrike line through the label — editorial "redacted"
+   cue, gentler than strikethrough. Drawn as a pseudo-element so it doesn't
+   shift typographic metrics. */
+.bh-layer-cell span::before {
+    content: "";
+    position: absolute;
+    left: 8%; right: 8%;
+    top: calc(50% - 1px);
+    height: 1px;
+    background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)